src/Templates/src/templates/maui-blazor/MauiApp.1.csproj

```diff
@@ -31,6 +31,9 @@
         <ApplicationDisplayVersion>1.0</ApplicationDisplayVersion>
         <ApplicationVersion>1</ApplicationVersion>
 
+        <!-- To create a packaged Windows application delete the nextg line and update your launch settings.  See https://aka.ms/MauiTemplateUnpackaged. -->
+        <WindowsPackageType>None</WindowsPackageType>
```

Comment:
We could potentially condition this behind something, like a solution configuration

Reply:
An MSBuild default like this would be nice:
```xml
<WindowsPackageType Condition=" '$(WindowsPackageType )' = '' and '$(Configuration)' == 'Debug' ">None</WindowsPackageType>
```

And then if customers wanted to opt out, they could put in their project:
```xml
<WindowsPackageType>Msix</WindowsPackageType>
```

But then `Properties/launchSettings.json` seems problematic...